packages/patternfly-4/react-core/src/components/Nav/NavExpandable.js

```diff
@@ -36,6 +37,52 @@ const defaultProps = {
   id: ''
 };
 
+
+class DelayedHiddenChild extends React.Component {
```

Comment:
Delaying the hidden doesn't really cause this to Expand/Collapse it just fades the children out. It might be better to find a way to actually transition the open/close by using height/max-height Here is a pretty good write up on how to do it with vanilla js.